CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -20,7 +20,10 @@ public class Employee
         public string? Position { get; set; }
 
         [ForeignKey(nameof(Company))]
```

Comment:
The comment '// TODO: Remove before merging' indicates temporary code left in the domain model. Leaving such comments in production code is error-prone and unprofessional.

```suggestion
Remove the comment and associated temporary field before merging to maintain code quality and avoid accidental deployment of test code.
```